codecov.yml

```diff
@@ -6,6 +6,12 @@ comment:
 github_checks:
   annotations: false
 
+codecov:
+  status:
+    patch:
+      default:
+        target: 33%
+
```

Comment:
I just updated some translated content, but it required me to upgrade more than 10% of the codecov/patch target, which I find incredible.

Reply:
I think I need to update, at most, the unit tests related to those files I changed, and not the unit tests of many unrelated files.